nixui/options/attribute.py

```diff
@@ -1,17 +1,17 @@
-import csv
 from dataclasses import dataclass, field
 import re
 from typing import List
-
+import csv
 
 @dataclass(frozen=True)
 class Attribute:
     loc: List[str] = field(default_factory=list)
 
-    @classmethod
-    def from_string(cls, attribute_string):
-        loc = next(csv.reader([attribute_string], delimiter='.', quotechar='"'))
-        return cls(loc)
+    def __init__(self, path):
+        if isinstance(path, list):
+            object.__setattr__(self, 'loc', path)
+        elif isinstance(path, str):
+            object.__setattr__(self, 'loc', next(csv.reader([path], delimiter='.', quotechar='"')))
```

Comment:
Recommend raising `TypeError` here. 

Looks good otherwise, nice work!

Reply:
Fixed this